dspace/config/modules/versioning.cfg

```diff
@@ -3,7 +3,6 @@
 #---------------------------------------------------#
 #  These configs are used by the versioning system  #
 #---------------------------------------------------#
-#Parameter 'enabled' is used only by JSPUI
```

Comment:
this parameter is still used by JSPUI only. To activate versioning for XMLUI the xmlui.xconf has to be changed. So please leave this comment as it was.